gluten-substrait/src/main/scala/org/apache/gluten/expression/ExpressionConverter.scala

```diff
@@ -109,6 +109,66 @@ object ExpressionConverter extends SQLConfHelper with Logging {
     }
   }
 
+  private def replaceStaticInvokeWithExpressionTransformer(
+      i: StaticInvoke,
+      attributeSeq: Seq[Attribute],
+      expressionsMap: Map[Class[_], String]): ExpressionTransformer = {
+    val objectName = i.staticObject.getName.stripSuffix("$")
+
+    if (objectName.endsWith("UrlCodec")) {
```

Comment:
Please use this format to simplify the code, and easy to implement others
```
val staticInvokeMap = Map("UrlCodec"-> Map("decode" -> ExpressionNames.URL_DECODE, "encode"-> ExpressionNames.URL_ENCODE ))
 val transformer = staticInvokeMap.filter(o => objectName.endsWith(o._1))
if (transformer.isEmpty) {
      throw
    }
```

Reply:
I think most of the cases we need all the argument, UrlCodec is a special case, maybe because the backend only supports "UTF-8" , we should throw exception if 2nd argument is not "UTF-8"  but not.

So other common cases should use the `staticInvokeMap`, that can also help developer figure out the special case.